pkgs/top-level/all-packages.nix

```diff
@@ -20372,7 +20372,13 @@ with pkgs;
 
   icon-lang = callPackage ../development/interpreters/icon-lang { };
 
-  libgit2 = callPackage ../development/libraries/libgit2 {
+  # Should be flipped for version 1.8.x soon
+  libgit2 = callPackage ../development/libraries/libgit2/1-7-2.nix {
```

Comment:
Don't flip back the default, just pin packages requiring an old version.

Reply:
That would mean this has to go to staging, which would mean that more Rust stuff will be broken after the 1.80 merge in a few days. We should not flip the default immediately.

Reply:
@SuperSandro2000 Plan is to do exactly that but in next PR that will go to staging. This one is just first step to unblock some merges to master in meantime.